packages/react-components/react-nav-preview/src/components/Nav/useNav.ts

```diff
@@ -83,5 +92,62 @@ export const useNav_unstable = (props: NavProps, ref: React.Ref<HTMLDivElement>)
     onUnregister,
     onSelect,
     getRegisteredNavItems,
+    onRequestNavCategoryItemToggle,
+    openItems,
   };
 };
+
+// /**
+//  * Initial value for the uncontrolled case of the list of open indexes
+//  */
+// function initializeUncontrolledOpenItems({ defaultOpenItems }: Pick<NavProps, 'defaultOpenItems'>): NavItemValue[] {
+//   if (defaultOpenItems !== undefined) {
+//     if (Array.isArray(defaultOpenItems)) {
+//       return [defaultOpenItems[0]];
+//     }
+//     return [defaultOpenItems];
+//   }
+//   return [];
+// }
+
+// /**
+//  * Normalizes Accordion index into an array of indexes
+//  */
+// function normalizeValues(index?: NavItemValue | NavItemValue[]): NavItemValue[] | undefined {
+//   if (index === undefined) {
+//     return undefined;
+//   }
+//   return Array.isArray(index) ? index : [index];
+// }
+
+// temp implementation of the above function.
+function normalizeValues(index?: NavItemValue | NavItemValue[]): NavItemValue[] | undefined {
+  return undefined;
+}
+
+/**
+ * Updates the list of open indexes based on an index that changes
+ * @param value - the index that will change
+ * @param previousOpenItems - list of current open indexes
+ * @param multiple - if Nav support multiple SubItemGroups opened at the same time
+ * @param collapsible - if Nav support multiple SubItemGroups closed at the same time
+ */
+function updateOpenItems(
+  value: NavItemValue,
+  previousOpenItems: NavItemValue[],
+  multiple: boolean,
+  collapsible: boolean,
```

Comment:
It seems this function is only called in one place, and `multiple` and `collapsible` are both always false. Is the plan to add props for them in the future? If not, can they be removed?

Reply:
Yep, my plan is to use them, but will be constantly re-evaluating if not.